src/Dev/BuildTask.php

```diff
@@ -62,9 +70,17 @@ abstract public function run($request);
     /**
      * @return bool
      */
-    public function isEnabled()
+    public function isEnabled(): bool
```

Comment:
```suggestion
    public function isEnabled()
```
We can't change method signatures unless we're doing a new major version i.e. CMS 6